fix: clarification, "using the OpenShift web console" (#2326)
codenvy/che-docs: modules/administration-guide/partials/proc_enabling-and-exposing-che-metrics.adoc

+
[NOTE]
====

> When {prod} is installed from the Red Hat Ecosystem Catalog using the default `CheCluster` CR, `spec.metrics.enable` is set to `true` by default. For example:

Reviewer: Guidelines: 
- Avoid using the passive voice.
- Following objects between backticks, name the object type.

Sometimes, the phrase, "by default," is unnecessary. I recommend using it only to describe pre-set parameter values or unmodified files. 

```suggestion
Installing {prod} from the Red Hat Ecosystem Catalog by using the default `CheCluster` CR sets the `spec.metrics.enable` parameter to `true`. For example:
```

Reviewer: If you want to emphasize that using the default `CheCluster` CR produces this outcome, as opposed to the outcome from using other possible installation methods, then move it up to the front of the sentence:
```suggestion
Using the default `CheCluster` CR to install {prod} from the Red Hat Ecosystem Catalog sets the `spec.metrics.enable` parameter to `true`.
```

Author: The when clause became irrelevant.

> When {prod} is installed from the Red Hat Ecosystem Catalog using the default `CheCluster` CR, `spec.metrics.enable` is set to `true` by default. For example:

Author: ```suggestion
`spec.metrics.enable` is set to `true` by default. For example:
```
